wenet/transformer/asr_model.py

```diff
@@ -20,6 +20,10 @@
 
 from torch.nn.utils.rnn import pad_sequence
 
+import k2
```

Comment:
move it into `hlg_onebest` & `hlg_rescore`, so we can run it without k2 & icefall.

Reply:
To avoid duplicate code, we use move these import into try except